src/rpc/client/ts/tests/fixtures.ts

```diff
@@ -60,14 +65,12 @@ export async function mochaGlobalSetup(): Promise<number | undefined> {
     fn: 'mochaGlobalSetup',
     msg: 'starting server',
     port: testPort,
-    pidfile: getPidFile(testPort),
   })
 
   // don't fix viewport data length in tests
   setAutoFixViewportDataLength(false)
 
   const pid = await startServer(rootPath, getClientVersion(), testPort)
-  mochaGlobalTeardown()
```

Comment:
We still want this too.  The scenario is when we kill a test, say with CTRL-C (or otherwise), this allows us to cleanup a server before we start it up again.

Reply:
We can't do this as it will kill the server we just started.

Reply:
Put it _before_ the startServer. :)